README.md

```diff
@@ -46,35 +46,88 @@ application you can leave away the `poetry run` part of the commands given below
 
 1. To launch the program run `poetry run python -m app_webcam` in the root of this repository
 2. Enjoy
-3. Press `q` to terminate the program
+3. Press `q` or simply click the close button to terminate the program
 
-## Architecture Overview
+## Component Diagram
+
+```mermaid
+    graph TD
+        
+          IP
+          
+          PF
+          
+          subgraph Output Processes
+            TTS --> TTS_ON[TTS Online]
+            TTS --> TTS_OFF[TTS Offline]
+            
+            PL --> CSV_L[CSV Logger]
+            PL --> CSL[Console Logger]
+            
+            VIS ~~~ END:::hidden
```

Comment:
what is this? this looks wrong as we now have a box on the diagram with the name "END"

Reply:
it was an attempt to push the 'VIS' to a desired place